src/services/userService.ts

```diff
@@ -0,0 +1,55 @@
+import crypto from 'crypto';
+import { BaseUser, User } from '../types/user.js';
+import { usersArray } from '../storage/localUsersStorage.js';
+
+
+class UserService {
+    private static generateUUID(): string {
+        return crypto.randomUUID();
```

Comment:
you can move this even to `utils` module as it is useful and applicable for the rest resources not only for User